src/interventions/android-webview-54.js

```diff
@@ -42,6 +42,6 @@ module.exports = (window, navigator) => {
 
   const regexResult = regexCheck.exec(userAgent);
   if (regexResult !== null) {
-    delete window.PaymentRequest;
+    window.PaymentRequest = null;
```

Comment:
Hi, 

The WooCommerce impl of PaymentRequest checks for PR support by doing 

'PaymentRequest' in window

which would imply that this solution is inefficient (that check will still be true). I would keep the "delete window.PaymentRequest" solution.

WooCommerce link: https://github.com/woocommerce/woocommerce-gateway-stripe/blob/master/assets/js/payment-request.js#L31